OpenRA.Mods.Common/FileFormats/InstallShieldCABCompression.cs

```diff
@@ -348,7 +383,19 @@ public InstallShieldCABCompression(Stream header, Dictionary<int, Stream> volume
 			if (header.ReadUInt32() != 0x28635349)
 				throw new InvalidDataException("Not an Installshield CAB package");
 
-			header.Position += 8;
+			var versionTmp = header.ReadUInt32();
+
+			// Logic taken from UnShield
```

Comment:
Include a link to https://github.com/twogood/unshield/blob/1.5.1/lib/libunshield.c#L277-L288 for completeness?